.github/workflows/tests.yml

```diff
@@ -31,7 +32,10 @@ jobs:
           TERM: linux # fix tput for tty issue work around
         run: |
           bash --version
-          bash -c "time ${{ matrix.env_vars }} bin/bats --formatter tap test"
+          bash -c "time ${{ matrix.env_vars }} bin/bats --log --formatter tap test" || EC=$?
+          [[ -f ./bats.log ]] && cat ./bats.log
```

Comment:
I'd rather have this as a test Case in test/bats.bats

Reply:
I can add tests once we can agree on what this should look like. I'd rather not spend the time just to have to change them :)